packages/common/src/store/ui/edit-access-confirmation-modal/sagas.ts

```diff
@@ -0,0 +1,20 @@
+import { takeEvery } from 'redux-saga/effects'
```

Comment:
is there a reason you're not using the `createModal` hook?

Reply:
i never know what pattern to use! is `createModal` the canonical one?

Reply:
good. think i'll leave this one as-is and we should probably do a modal revamp at some point as a team.